### PR TITLE
remove unnessary bundle listener triger when topic lookup

### PR DESCRIPTION
**Pull request by hangc0276**
### Motivation
When call `resolveOwnership`, it will trigger listener notify, and the registered listeners will trigger onLoad/unLoad operation. However, the `resolveOwnership` will be called by topic lookup check, thus the registered listeners will be notified on each lookup check.  In fact, the listeners only need to be notified when bundle load/unload.

### Changes
1. remove listeners' trigger notify operation on `resolveOwnership`

### CODE REVIEW

**Comment by hangc0276** [comment]
@kezhuw  Would you please help review the code?

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceOwnershipListenerTests.java [hunk 1]
```diff
@@ -124,4 +125,53 @@ public void testGetAllPartitions() throws PulsarAdminException, ExecutionExcepti
         admin.namespaces().deleteNamespace(namespace);
     }
 
+    @Test
+    public void testNamespaceBundleLookupOnwershipListener() throws PulsarAdminException, InterruptedException,
```
Comment (kezhuw) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceOwnershipListenerTests.java:129]: Typo, should be`Ownership`.

Comment (kezhuw) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceOwnershipListenerTests.java:129]: The test does not fail either without changes applied.

**Review by kezhuw — COMMENTED** [review]
Hi @hangc0276, thanks for reaching out to me. The major problem of this issue is that it breaks in corner cases(eg. dangling ownership acquiring/releasing, see #7517). I did not add assertions for namespace bundle listener in #7773 though. I just pushed a branch with [assertions](https://github.com/kezhuw/pulsar/commit/efcc6566ff1e3c417103827f561ec7dcc496f9bc) to guard this. This pr breaks `TopicOwnerTest.testAcquireOwnershipWithZookeeperDisconnectedAfterOwnershipNodeCreated` and `TopicOwnerTest.testReleaseOwnershipWithZookeeperDisconnectedBeforeOwnershipNodeDeleted` in that branch. I guess it is worth to add these assertions for guarding. I will open issue to talk about this later days.

**Review by codelipenghui — APPROVED** [review]

**Review by jiazhai — APPROVED** [review]
